bookwyrm/templates/book/edit/edit_book_form.html

```diff
@@ -28,6 +28,15 @@ <h2 class="title is-4">
                     {% include 'snippets/form_errors.html' with errors_list=form.title.errors id="desc_title" %}
                 </div>
 
+                <div class="field">
+                    <label class="label" for="id_sort_title">
+                        {% trans "Sort Title:" %}
+                    </label>
+                    <input type="text" name="sort_title" value="{{ form.sort_title.value|default:'' }}" maxlength="255" class="input" required="" id="id_sort_title" aria-describedby="desc_sort_title">
+
+                    {% include 'snippets/form_errors.html' with errors_list=form.sort_title.errors id="desc_sort_title" %}
+                </div>
```

Comment:
I added the sort title to the edit book form.  I wonder if more context is needed for users to understand what a sort title is used for?  For example I could imagine an information icon next to the label with a description that appears in a tooltip on hover?  I could also imagine this hidden in an expandable accordion type ui element under "advanced" in order to keep this form from being too cluttered?

![Screen Shot 2023-04-26 at 11 02 44 PM](https://user-images.githubusercontent.com/2885925/234757384-37d91681-b342-491f-97a5-b7d15caaf573.png)

Reply:
I do think this (and a number of other fields) could use some clarification, but I think it's outside the scope of this PR, since I want to make sure we approach it with consistent UI and markup